--- "lua-language-server-3.18.2/script/config/template.lua"
+++ "faf-lua-language-server-patch/script/config/template.lua"
@@ -185,6 +185,7 @@ end)
 ---@format disable-next
 local template = {
     ['Lua.runtime.version']                 = Type.String >> 'Lua 5.4' << {
+                                                'LuaFA',
                                                 'Lua 5.1',
                                                 'Lua 5.2',
                                                 'Lua 5.3',
@@ -228,6 +229,9 @@ local template = {
                                             }),
     ['Lua.runtime.plugin']                  = Type.Or(Type.String, Type.Array(Type.String)) ,
     ['Lua.runtime.pluginArgs']              = Type.Or(Type.Array(Type.String), Type.Hash(Type.String, Type.String)),
+    -- FAForever: when true, files that set globals will export env by default;
+    -- use ---@declare-global to opt-out or ---@export-env to opt-in when false
+    ['Lua.runtime.exportEnvDefault']        = Type.Boolean >> false,
     ['Lua.runtime.fileEncoding']            = Type.String >> 'utf8' << {
                                                 'utf8',
                                                 'ansi',
