--- luals-3.18.2/script/files.lua	2026-06-20 07:50:37.136349077 +0000
+++ luals-3.18.2-patched/script/files.lua	2026-06-20 07:50:54.710789261 +0000
@@ -598,9 +598,10 @@
 
     ---@type brave.param.compile.options
     local options = {
-        special           = config.get(uri, 'Lua.runtime.special'),
-        unicodeName       = config.get(uri, 'Lua.runtime.unicodeName'),
-        nonstandardSymbol = util.arrayToHash(config.get(uri, 'Lua.runtime.nonstandardSymbol')),
+        special            = config.get(uri, 'Lua.runtime.special'),
+        unicodeName        = config.get(uri, 'Lua.runtime.unicodeName'),
+        nonstandardSymbol  = util.arrayToHash(config.get(uri, 'Lua.runtime.nonstandardSymbol')),
+        exportEnvDefault   = config.get(uri, 'Lua.runtime.exportEnvDefault'),
     }
 
     ---@type brave.param.compile
@@ -652,9 +653,10 @@
 
     ---@type brave.param.compile.options
     local options = {
-        special           = config.get(uri, 'Lua.runtime.special'),
-        unicodeName       = config.get(uri, 'Lua.runtime.unicodeName'),
-        nonstandardSymbol = util.arrayToHash(config.get(uri, 'Lua.runtime.nonstandardSymbol')),
+        special            = config.get(uri, 'Lua.runtime.special'),
+        unicodeName        = config.get(uri, 'Lua.runtime.unicodeName'),
+        nonstandardSymbol  = util.arrayToHash(config.get(uri, 'Lua.runtime.nonstandardSymbol')),
+        exportEnvDefault   = config.get(uri, 'Lua.runtime.exportEnvDefault'),
     }
 
     local ws     = require 'workspace'
