--- "lua-language-server-3.18.2/script/core/completion/completion.lua"
+++ "faf-lua-language-server-patch/script/core/completion/completion.lua"
@@ -489,7 +489,7 @@ local function checkFieldFromFieldToIndex(state, name, parent, word, position)
             }
         end
     else
-        if config.get(state.uri, 'Lua.runtime.version') == 'Lua 5.1'
+        if (config.get(state.uri, 'Lua.runtime.version') == 'Lua 5.1' or config.get(state.uri, 'Lua.runtime.version') == 'LuaFA')
         or config.get(state.uri, 'Lua.runtime.version') == 'LuaJIT' then
             textEdit.newText = '_G' .. textEdit.newText
         else
@@ -1782,6 +1782,8 @@ local function tryluaDocCate(word, results)
         'overload',
         'deprecated',
         'meta',
+        'export-env',
+        'declare-global',
         'version',
         'see',
         'diagnostic',
