--- "lua-language-server-3.18.2/script/files.lua"
+++ "faf-lua-language-server-patch/script/files.lua"
@@ -601,6 +601,7 @@ function m.compileStateAsync(uri, callback)
         special           = config.get(uri, 'Lua.runtime.special'),
         unicodeName       = config.get(uri, 'Lua.runtime.unicodeName'),
         nonstandardSymbol = util.arrayToHash(config.get(uri, 'Lua.runtime.nonstandardSymbol')),
+        exportEnvDefault   = config.get(uri, 'Lua.runtime.exportEnvDefault'),
     }
 
     ---@type brave.param.compile
@@ -655,6 +656,7 @@ function m.compileState(uri)
         special           = config.get(uri, 'Lua.runtime.special'),
         unicodeName       = config.get(uri, 'Lua.runtime.unicodeName'),
         nonstandardSymbol = util.arrayToHash(config.get(uri, 'Lua.runtime.nonstandardSymbol')),
+        exportEnvDefault   = config.get(uri, 'Lua.runtime.exportEnvDefault'),
     }
 
     local ws     = require 'workspace'
