--- luals-3.18.2/script/brave/work.lua	2026-06-20 07:50:37.117677622 +0000
+++ luals-3.18.2-patched/script/brave/work.lua	2026-06-20 07:50:54.698425828 +0000
@@ -107,6 +107,7 @@
 ---@field special table<string, string>
 ---@field unicodeName boolean
 ---@field nonstandardSymbol table<string, true>
+---@field exportEnvDefault boolean
 
 ---@param param brave.param.compile
 brave.on('compile', function (param)
