--- "lua-language-server-3.18.2/script/parser/luadoc.lua"
+++ "faf-lua-language-server-patch/script/parser/luadoc.lua"
@@ -1364,6 +1364,22 @@ local docSwitch = util.switch()
         meta.name = parseName('doc.meta.name', meta)
         return meta
     end)
+    : case 'export-env'
+    : call(function ()
+        return {
+            type   = 'doc.export-env',
+            start  = getFinish(),
+            finish = getFinish(),
+        }
+    end)
+    : case 'declare-global'
+    : call(function ()
+        return {
+            type   = 'doc.declare-global',
+            start  = getFinish(),
+            finish = getFinish(),
+        }
+    end)
     : case 'version'
     : call(function ()
         local result = {
