--- luals-3.18.2/script/provider/provider.lua	2026-06-20 07:50:37.150827156 +0000
+++ luals-3.18.2-patched/script/provider/provider.lua	2026-06-20 07:50:55.027503755 +0000
@@ -272,6 +272,12 @@
     ---@async
     function (params)
         local doc      = params.textDocument
+        -- FAForever: honour supportScheme whitelist
+        local scheme   = furi.split(doc.uri)
+        local supports = config.get(doc.uri, 'Lua.workspace.supportScheme')
+        if supports and #supports > 0 and not util.arrayHas(supports, scheme) then
+            return
+        end
         local uri      = files.getRealUri(doc.uri)
         log.debug('didOpen', uri)
         local text  = doc.text
@@ -301,6 +307,12 @@
     function (params)
         local fixIndent = require 'core.fix-indent'
         local doc     = params.textDocument
+        -- FAForever: honour supportScheme whitelist
+        local scheme   = furi.split(doc.uri)
+        local supports = config.get(doc.uri, 'Lua.workspace.supportScheme')
+        if supports and #supports > 0 and not util.arrayHas(supports, scheme) then
+            return
+        end
         local changes = params.contentChanges
         local uri     = files.getRealUri(doc.uri)
         local text    = files.getOriginText(uri)
